### PR TITLE
docs(scully): add scully's CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Villanuevand**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+     ```
+
+1. Create your patch, **including appropriate test cases**.
+
+1. Commit your changes using a descriptive commit message that follows our
+  [commit message conventions](#commit). Adherence to these conventions
+  is necessary because release notes are automatically generated from these messages.
+
+     ```shell
+     git commit -a
```
Comment (SanderElias) [inline CONTRIBUTING.md]: this is for contributing to scully.
don't adviise got commit, but npm run commit instead

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+## <a name="commit"></a> Commit Message Guidelines
+
+We have very precise rules over how our git commit messages can be formatted.  This leads to **more
+readable messages** that are easy to follow when looking through the **project history**.
+
+### Commit Message Format
+Each commit message consists of a **header**, a **body** and a **footer**.  The header has a special
+format that includes a **type**, a **scope** and a **subject**:
+
```
Comment (SanderElias) [inline CONTRIBUTING.md]: mention something about `npm run commit` here too.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Villanuevand) [reply CONTRIBUTING.md]: You're totally right ...

File: CONTRIBUTING.md (hunk 2 above)
Comment (Villanuevand) [reply CONTRIBUTING.md]: Done!

File: CONTRIBUTING.md (hunk 1 above)
Comment (Villanuevand) [reply CONTRIBUTING.md]: Change added in [#fe333b2](https://github.com/scullyio/scully/pull/63/commits/ce850bb23fd4ee36a0d2d58cf5803d226fe333b2)

File: CONTRIBUTING.md (hunk 2 above)
Comment (Villanuevand) [reply CONTRIBUTING.md]: Added!

Comment (Villanuevand) [reply CONTRIBUTING.md]: Change added in [#fe333b2](https://github.com/scullyio/scully/pull/63/commits/ce850bb23fd4ee36a0d2d58cf5803d226fe333b2)

**Pull request merged**